src/main/java/uk/gov/hmcts/divorce/citizen/event/CitizenSwitchedToSole.java

```diff
@@ -74,12 +75,14 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
             data.getCaseInvite().getApplicant2UserId()
         );
 
+        applicant1SwitchToSoleNotification.sendToApplicant1(data, details.getId());
+        if (data.getApplication().getApplicant2ScreenHasMarriageBroken() != YesOrNo.NO) {
```

Comment:
should be .equals ?

Reply:
@nitinprabhu I tried using .equals but it throws a nullPointerException when the applicant2ScreenHasMarriageBroken is null, which is possible for this event